pkg/cloudprovider/providers/aws/cloudformation_templates_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2017 The Keto Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"testing"
+
+	"github.com/UKHomeOffice/keto/pkg/model"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+func TestRenderClusterInfraStackTemplate(t *testing.T) {
+	subnets := []*ec2.Subnet{
+		{
+			SubnetId:         aws.String("subnet0"),
+			AvailabilityZone: aws.String("az0"),
+		},
+		{
+			SubnetId:         aws.String("subnet1"),
+			AvailabilityZone: aws.String("az1"),
+		},
+	}
+
+	_, err := renderClusterInfraStackTemplate("foo", "vpc123", subnets)
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
… and all the other test cases in this file. Seems they just test that no error is returned? Maybe test the outputs too? (as well as any expectations on dependencies, if relevant).